docs/articles/toc.yml

```diff
@@ -1,66 +1,76 @@
 - name: Introduction
   href: intro.md
-- name: Getting started
+- name: The Basics
   items:
-  - name: Bot application
-    href: getting_started/bot_application.md
-  - name: Basic bot
-    href: getting_started/basic_bot.md
-  - name: Alternate WebSocket clients
-    href: getting_started/alternate_ws.md
-  - name: Event reference
-    href: getting_started/event_reference.md
-  - name: Notes about using Mono
-    href: getting_started/mono.md
+  - name: Creating a Bot Application
+    href: basics/bot_application.md
+  - name: Writing a Basic Bot
+    href: basics/basic_bot.md
+- name: Beyond Basics
+  items:
+  - name: Understanding TAP Events
+    href: beyond_basics/event_reference.md
+  - name: Alternate WebSocket Clients
+    href: beyond_basics/alternate_ws.md
+  - name: Using the Mono Runtime
+    href: beyond_basics/mono.md
+  - name: Sharding
+    href: beyond_basics/sharding.md
 - name: Commands
   items:
   - name: Introduction
     href: commands/intro.md
-  - name: Argument converters
+  - name: Argument Converters
     href: commands/argument_converters.md
-  - name: Dependency injection
-    href: commands/dependency_injection.md
-  - name: Module lifespans
+  - name: Module Lifespans
     href: commands/module_lifespans.md
-  - name: Custom command handlers
+  - name: Dependency Injection
+    href: commands/dependency_injection.md
+  - name: Custom Command Handler
     href: commands/custom_handler.md
+  - name: Command Attributes
+    href: commands/command_attributes.md
 - name: Interactivity
   items:
   - name: Introduction
     href: interactivity/intro.md
-- name: Voice
+- name: Audio
```

Comment:
I'm not so sure I like this reorganization. If anything, I'd definitely place Lavalink before VoiceNext, as it will satisfy 90% of the userbase.